packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'theme.dart';
+import 'typography.dart';
+
+class TimeOfDay {
+  const TimeOfDay({ this.hour, this.minute });
+
+  /// The selected hour, in 24 hour time.
```

Comment:
You should specify the range is 0..23, so that Norwegians don't ask you if "24:00" is a valid time.